src/middleware/session.middleware.ts

```diff
@@ -13,3 +14,9 @@ const cookieConfig: CookieConfig = {
 };
 
 export const sessionMiddleware = SessionMiddleware(cookieConfig, sessionStore, true);
+
+export const csrfProtectionMiddleware = CsrfProtectionMiddleware({
```

Comment:
Wouldn't it be more sensible to move this to its own file instead of mixing it with the session middleware? At least this what I'm doing in your-companies-web

Reply:
Yeah, we can move it into its own file and just export the session store from the session middleware.